src/lib.rs

```diff
@@ -169,6 +169,16 @@ impl std::fmt::Debug for Window {
 #[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
 pub struct WindowId(platform::WindowId);
 
+impl WindowId {
+    /// Returns a dummy WindowId, useful for unit testing. Absolutely no guarantees are made
+    /// about this window id, it may be equal to a real WindowId.
+    ///
+    /// **Passing this into a winit function will result in undefined behavior.**
```

Comment:
It doesn't look like Winit has any publicly-facing functions that take `WindowId`s, so this warning doesn't currently apply anywhere. However, this function should be marked unsafe for forwards-compatibility reasons since it could cause undefined behavior issues if we *do* introduce such functions. Once we approach 1.0, we can look at making this safe if we know we aren't going to introduce other functions that make it unsafe.